client/client.lua

```diff
@@ -56,13 +57,22 @@ RegisterKeyMapping('createscene', 'Create Scene', 'keyboard', Config.CreateScene
 RegisterKeyMapping('deletescene', 'Delete Scene', 'keyboard', Config.DeleteSceneKey)
 
 RegisterCommand('createscene', function()
+    Wait(50)
+    if (Config.AdminOnly and permissionLevel == 'user') then 
+        TriggerEvent('QBCore:Notify', 'Must be an Admin to use create laser', 'error')
+        return 
+    end
     OpenMenu()
     TriggerServerEvent("InteractSound_SV:PlayOnSource", "monkeyopening", 0.05)
 end)
 
-RegisterCommand('deletescene', function()
```

Comment:
This breaks the deletescene hotkey, why was this changed to an nuicallback when it already exists below?